extensions/amp-sidebar/0.1/amp-sidebar.js

```diff
@@ -80,13 +88,15 @@ export class AmpSidebar extends AMP.BaseElement {
       this.element.setAttribute('aria-hidden', 'true');
     }
 
+    // Prepare for animation (will not animate the initial state).
+    this.element.classList.add('-amp-sidebar-animate');
```

Comment:
I think you are correct - removing the animate class - as it is not required after the vsync changes.